.github/actions/nss/action.yml

```diff
@@ -1,19 +1,53 @@
 name: Fetch and build NSS
 
+inputs:
+  type:
+    description: 'Whether to do a debug or release build of NSS'
+    required: true
+    default: 'Debug'
```

Comment:
Having both required and default seem to be incompatible.

I would also recommend a default of 'Release' on the basis that we won't be debugging NSS code in CI, so the extra performance is probably worth it.